_episodes/15-coding-conventions.md

```diff
@@ -71,10 +71,22 @@ a description of a new feature in Python, etc.
 {: .callout}
 
 As we have already covered in the
-[episode on PyCharm IDE](../13-ides/index.html),
-PyCharm highlights the language constructs (reserved words)
-and syntax errors to help us with coding.
-PyCharm also gives us recommendations for formatting the code -
+[episode on VS Code IDE](../13-ides/index.html),
+VS Code highlights the language constructs (reserved words)
+and syntax errors to help us with coding. We would also like VS Code to give us recommendations 
+for formatting the code (for example using 
+the PEP 8 style guide above). To do this we will need to install another extension. 
+(See the [episode on VS Code IDE](../13-ides/index.html) if you need a reminder on how to add extensions.)
+On 'Activity Bar', on the left hand side of the VS Code interface, click on the `Extensions` Tab,
+so that we can add the an extension called `Flake8` for VS Code.
+In the search text box, type `Flake8`. Select the `Flake8` extension, as shown in the following figure,
+and click the blue `install` button. Note that other Python style guide extensions are available. 
+Another useful one at the time of writing, is one called `Pylint`.
```

Comment:
Let's mention Ruff as well, as suggested by @amangoel185 https://github.com/carpentries-incubator/python-intermediate-development-earth-sciences/issues/40

Reply:
Will do that next time I'm on my Linux laptop.